lib/profiles.js

```diff
@@ -108,6 +108,27 @@ class Profiles extends Private {
             data: params
         });
     }
+
+    /**
+     * Unset properties on a given profile
+     * @param {string} profileId The Klaviyo generated ID for a profile.
+     * @param {list} properties The list of properties to unset.
+     * @returns {Promise<Object|KlaviyoApiError>} Updated profile data | Non-200 response received.
+     */
+    unsetProfileProperties({ profileId, properties=[] } = {}) {
+        if (!profileId) { throw new KlaviyoError('Profile ID was not provided'); }
+        if (properties.length < 1) { throw new KlaviyoError('No properties provided.'); }
```

Comment:
Should we validate that this is an array? We don't really return any info if a request doesn't work e.g. if you just pass in a simple string